### PR TITLE
Fix #388 issue

### LINKED ISSUE #388: Certain messages crash in graphql_to_extensible_attachment
## Description of the problem
Retrieving certain messages throws an exception. I unfortunately am not sure which messages these are, but they seem to be about one out of every five thousand.

## Traceback
```
[Lots of lines of traceback falling down to]
    ms = self.client.fetchThreadMessages(thread, lim, before=self.laststamp)
  File "/usr/local/lib/python3.6/dist-packages/fbchat/client.py", line 1105, in fetchThreadMessages
    for message in j["message_thread"]["messages"]["nodes"]
  File "/usr/local/lib/python3.6/dist-packages/fbchat/client.py", line 1105, in <listcomp>
    for message in j["message_thread"]["messages"]["nodes"]
  File "/usr/local/lib/python3.6/dist-packages/fbchat/graphql.py", line 394, in graphql_to_message
    attachment = graphql_to_extensible_attachment(message["extensible_attachment"])
  File "/usr/local/lib/python3.6/dist-packages/fbchat/graphql.py", line 196, in graphql_to_extensible_attachment
    if story.get("media")
TypeError: 'NoneType' object is not subscriptable
```

## Environment information
- Python 3.6.7
- `fbchat` 1.6.1

### PR DESCRIPTION
**Pull request by kapi2289**

### CODE REVIEW

File: fbchat/graphql.py [hunk 1]
```diff
@@ -166,19 +167,18 @@ def graphql_to_extensible_attachment(a):
...
+                if story.get("media"):
+                    if story["media"].get("image"):
+                        rtn.image_url = story["media"]["image"]["uri"]
+                        rtn.image_width = story["media"]["image"]["width"]
+                        rtn.image_height = story["media"]["image"]["height"]
                 rtn.url = story["url"]
                 return rtn
             elif _type in ["ExternalUrl", "Story"]:
-                return ShareAttachment(
+                rtn = ShareAttachment(
```
Comment (madsmtm) [inline fbchat/graphql.py]: You forgot to actually return this!

File: fbchat/graphql.py [hunk 2]
```diff
@@ -191,28 +191,22 @@ def graphql_to_extensible_attachment(a):
...
-                    if story.get("media")
-                    else None,
+                    source=story["source"].get("text"),
                     attachments=[
                         graphql_to_subattachment(attachment)
                         for attachment in story.get("subattachments")
                     ],
                 )
+                if story.get("media"):
+                    if story["media"].get("image"):
```
Comment (madsmtm) [inline fbchat/graphql.py]: Merge the two if-statements with `and`

Comment (kapi2289) [reply fbchat/graphql.py]: Are sure this won't throw an error if `media` will be `None`? I tried this in Python console and it through an error.

File: fbchat/graphql.py (hunk 1 above)
Comment (kapi2289) [reply fbchat/graphql.py]: Hahah, indeed!

File: fbchat/graphql.py (hunk 2 above)
Comment (madsmtm) [reply fbchat/graphql.py]:
```py
>>> story = {}
>>> story.get('media') and story["media"].get("image")
None
```
Yes, I'm sure 😁. The expression behind `and` won't get executed unless the previous expression was `True`. The same goes for `or`. Another example:
```py
>>> False and invalid
False
>>> True or invalid
True
```

**Pull request merged**